local-process-quickstart.sh

```diff
@@ -70,8 +82,17 @@ while getopts "g:n:r:cd" opt; do
    esac
 done
 
+
+if [ -z "$BIKENS" ]; then
```

Comment:
Is there not an option for this? Do you just need to know to set `BIKENS` as an environment variable before calling the script?

Reply:
I did put an option in for it, but the behaviour was unusual.  I postponed continuing with the option for this PR.  Lets talk this one over.